.cargo/config

```diff
@@ -8,5 +8,11 @@ target-dir = "build/cargo_target"
 # No need for the `crt_static` flag as rustc appends it by default.
 rustflags = [
 	"-C", "link-arg=-lgcc",
+	"-D", "warnings",
 	"-C", "link-arg=-lfdt",
 ]
+
+[target.'cfg(not(any(target_arch="arm", target_arch="aarch64")))']
```

Comment:
This can easily become annoying when doing development. If you have code in flux and you want to test it, my understanding is that the build will fail locally. Can we instead have this as a separate test in the CI to check that when trying to push something upstream there are no build warnings?

Reply:
Can we do this in cargo for release builds only?

Reply:
I moved the check inside the integration tests. Take a look!